tests/integration/common/test_py_version_checking.py

```diff
@@ -43,6 +43,14 @@ def test_py_version_mismatch(capsys):
     base_image = 'registry.hub.docker.com/library/python:{}'.format(py_version)
     run_submission_with_function_preprocessor(capsys, base_image, False)
 
+def test_py_patch_version_match(capsys):
+    current_py_version = ".".join([str(x) for x in sys.version_info[0:2]])
+    if current_py_version == "3.6.5":
```

Comment:
That's strange, I cannot get the intergration logs, but I guess the new case failed. the case confused me, why get [0:2] but compile all version here?  and you expected the version matched? if yes, we need to pass `is_match=True` while calling `run_submission_with_function_preprocessor`

Reply:
Good catch. It was supposed to be [0:3] I added this one because in current implementation, we do compare micro version. Since we just compare [0:2] now, we don't need it anymore. I delete the test case and also update original test case. Please check commit and test pass now,